### PR TITLE
Patch/added

### PR DESCRIPTION
**Pull request by kennygperez**

### CODE REVIEW

**Review by Goff-Davis — CHANGES_REQUESTED** [review]
How are you handling more questions being added? What happens when you delete a question and then add a new one?

File: frontend/src/store.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ const init = {
   questionNumber: 0,
   classSize: 0,
   openExportModal: false,
-  currentQuestionInfo: JSON.parse(JSON.stringify(baseQuestionInfo)),
+  currentQuestionInfo: Object.assign({}, baseQuestionInfo),
```
Comment (Goff-Davis) [inline frontend/src/store.js:31]: This is faster because it is not a deep copy. Please revert pack to JSON.parse and JSON.stringifiy

**Comment by kennygperez** [comment]
> How are you handling more questions being added? What happens when you delete a question and then add a new one?

The way it was done before

**Review by Goff-Davis — CHANGES_REQUESTED** [review]

**Comment by Goff-Davis** [comment]
> > How are you handling more questions being added? What happens when you delete a question and then add a new one?
> 
> The way it was done before

The way it was done before does not support ids

**Pull request closed without merging**